### PR TITLE
Tweet utils (segment large text into tweets, post as thread)

### PR DESCRIPTION
**Pull request by maiamcc**
Lol it would be great if the API took care of breaking a string down into tweet-sized chunks for you, but if you try to tweet something that's too long you get back the error message:
```
tweepy.errors.Forbidden: 403 Forbidden
186 - Tweet needs to be a bit shorter.
```

This PR contains utils for (somewhat intelligently) breaking down a big string into tweet-sized chunks, and for posting multiple tweets as a thread (i.e. each in reply to the previous tweet).

(I wrote this PR assuming that we'd have big chunks of text we needed to send, e.g. if giving someone multiple options for what officer the meant. It occurs to me now that we could just decide that each "did you mean...?" could be its own tweet and we wouldn't need this slightly-too-fancy chunking logic. But uhhhh whoops it's here now!)

### CODE REVIEW

File: app/tweet_utils.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import re
+from typing import List
+
+import tweepy
+
+
+CHAR_LIMIT = 280
```
Comment (mNov) [inline app/tweet_utils.py:7]: i remember when it was 140!

File: test/test_tweet_utils.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        assert body_from_tweet(tweet) == "hello world"
+
+    def test_strips_leading_trailing_whitespace(self):
+        tweet = tweet_with_text("""
+          hello world
+""")
+        assert body_from_tweet(tweet) == "hello world"
+
+
+class TestChunkIntoTweets:
```
Comment (mNov) [inline test/test_tweet_utils.py:27]: very helpful to see this test class. thanks for adding it!

**Review by mNov — APPROVED** [review]
lgtm!

File: test/test_tweet_utils.py (hunk 2 above)
Comment (mNov) [reply test/test_tweet_utils.py:27]: just one suggestion, actually - could you add a test case where the `char_limit` is set to the default value, 280? i assume that's the only value that would be used outside these test cases? also, where is `chunk_into_tweets` called from? sorry if I'm missing it but I don't see any calling functions...

Comment (maiamcc) [reply test/test_tweet_utils.py:27]: I mean I can totally put that test in but I'm not sure what it's testing. Just that the default value is passed correctly? That seems a little silly—or am I missing something?

It's called in the next PR!

**Pull request merged**